editor.go

```diff
@@ -17,53 +17,90 @@ func NewEditor() *Editor {
 
 //go:generate gentut -t Editor -p owasp -in editor.go -w
 type Editor struct {
-	entries []Entry
+	Entries []Entry
+}
+
+// SetApplicableByLevel sets the all entries with the given level or lower as applicable
+func (me *Editor) SetApplicableByLevel(level string) error {
+	for i, e := range me.Entries {
+		if level == "L3" {
+			me.Entries[i].Applicable = true
+		} else if level == "L2" && e.L2 {
+			me.Entries[i].Applicable = true
+		} else if level == "L1" && e.L1 {
+			me.Entries[i].Applicable = true
+		} else {
+			me.Entries[i].Applicable = false
```

Comment:
This means that if I Use `editor.SetApplicableByLevel("no such level")`, every entry will be reset.
Since the levels are very static, named levels 
```go
type Level int
const (
 _ Level = iota
  L1
  L2
  L3
)
```

Then the level argument can be typed accordingly. I think it'll also read nicely, i.e.

`editor.SetApplicableByLevel(owasp.L1)`

Reply:
See `Report.groupByLevel()` which today uses a plain int.